annotation-processor/src/main/java/mekanism/builder/ComputerHandlerBuilder.java

```diff
@@ -0,0 +1,607 @@
+
+            //build the method
+            CodeBlock methodBody = CodeBlock.builder()
+                  .add(valueReturner)
+                  .build();
+            MethodSpec handlerMethod = buildHandlerMethod(annotatedName + "$" + targetMethodName, methodBody);
+            handlerTypeSpec.addMethod(handlerMethod);
+
+            //add a call to register() in the handler class's constructor
+            CodeBlock registerMethodBuilder = buildRegisterMethodCall(annotationValues, Collections.emptyList(), wrapperMethod.getReturnType(), handlerMethod, targetMethodName, annotationValues.getLiteral("threadSafe", false));
+            constructorBuilder.addStatement(registerMethodBuilder);
+
+        }
+    }
+
+    /**
+     * Get a list of public static methods of the wrapper type. They should have only one param, but this is not currently checked.
+     *
+     * @param wrapperTypeEl the wrapper type to check
+     *
+     * @return a list of method elements
+     */
+    private static List<ExecutableElement> gatherWrapperMethods(TypeElement wrapperTypeEl) {
+        List<ExecutableElement> wrapperMethods = new ArrayList<>();
+        for (Element element : wrapperTypeEl.getEnclosedElements()) {
+            if (element instanceof ExecutableElement exec) {
+                Set<Modifier> modifiers = exec.getModifiers();
+                if (modifiers.contains(Modifier.PUBLIC) && modifiers.contains(Modifier.STATIC)) {
+                    wrapperMethods.add(exec);
+                }
+            }
+        }
+        return wrapperMethods;
+    }
+
+    /**
+     * Generate a getter and/or setter for a field. Private fields do not support setter (currently unused anyway)
+     *
+     * @param annotatedName        the field or method simple name
+     * @param isPrivateOrProtected modifiers checked previously
+     * @param isStatic             modifier checked previously
+     * @param annotationValues     an annotation helper for this annotation's values
+     * @param fieldElement         the field annotated
+     */
+    private void handleAtSyntheticMethod(String annotatedName, boolean isPrivateOrProtected, boolean isStatic, AnnotationHelper annotationValues, VariableElement fieldElement) {
+        //get a code reference to the field, or a call of the generated getter
+        CodeBlock targetReference = getReadTargetReferenceForField(annotatedName, isPrivateOrProtected, isStatic, fieldElement);
+
+        //the type of the annotated field
+        TypeMirror fieldType = fieldElement.asType();
+
+        //generate a getter if needed
+        String getterName = annotationValues.getStringValue("getter", null);
+        if (getterName != null) {
+            //convert the field result
+            CodeBlock valueReturner = convertValueToReturn(fieldElement, fieldType, targetReference);
+            //add a method for it
+            MethodSpec handlerMethod = buildHandlerMethod(getterName + "_0", valueReturner);
```

Comment:
convention with the other methods - the generated method names have the number of params appended.
So this would prevent a synthetic getter and a normal computer method named the same from being generated